app/services/ocrService.py

```
@@ -19,25 +20,100 @@
+) -> str:
+    try:
+        bucket = _get_bucket()
+
+        # users/{user_hash}/ocr 경로로 저장 (파일명 ocr_{rec.record_date}_{timestamp})
+        destination_blob_name = get_user_ocr_path(user_hash, filename)
+        blob = bucket.blob(destination_blob_name)
+
+        blob.upload_from_string(file_bytes, content_type=content_type)
+        print(f"GCS 업로드 완료: gs://{GCS_BUCKET_NAME}/{destination_blob_name}")
+
+        return destination_blob_name
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)